src/wizards/generateProject/generationWizard.ts

```diff
@@ -58,14 +58,36 @@ export async function generateProjectWizard() {
     });
 
     state.buildTool = (await input.showQuickPick({
-      title: 'Quarkus Tools',
+      title: INPUT_TITLE,
       step: input.getStepNumber(),
       totalSteps: state.totalSteps,
       placeholder: 'Pick build tool',
       items: quickPickItems,
       activeItem: quickPickItems[0]
     })).label;
 
+    return (input: MultiStepInput) => inputPlatformVersion(input, state);
+  }
+
+  async function inputPlatformVersion(input: MultiStepInput, state: Partial<ProjectGenState>) {
+
+    const quickPickItems: PlatformVersionPickItem[] = await getCodeQuarkusApiPlatforms();
+
+    // Sort by recommended and version number in case of tie in the quick pick list
+    quickPickItems.sort((x: PlatformVersionPickItem, y: PlatformVersionPickItem) => {
+      return (x.recommended === y.recommended) ? (parseFloat(x.label) > parseFloat(y.label) ? -1 : 1) : x.recommended ? -1 : 1;
```

Comment:
I wouldn't use x.label, because it'll be something like "2.2 (recommended)" where recommended is true. You can just redefine `x["key"].split(":")[1]` and use the result of that.

Reply:
parseFloat("2.2 (recommended)") will give 2.2: https://www.typescriptlang.org/play?#code/MYewdgziA2CmB00QHMAUAHAhgJwrAYkpgC6oBEATPBQASraygC2TsYAJrO2QJQ9A

Reply:
Ok, let's leave it as is. Turns out this is well described behaviour https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat .